github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits.yaml

```diff
@@ -31,6 +31,37 @@ presubmits:
           resources:
             requests:
               memory: "8Gi"
+  - name: pull-cdi-unit-test-s390x
+    skip_branches:
+      - release-v1.38
+      - release-v1.43
+      - release-v1.49
+    cluster: s390x-virtual-workloads #
```

Comment:
Could we use the same node that runs the kubevirt unit tests? 

https://github.com/kubevirt/project-infra/blob/15d3b253c6484d75f396700b16093a58a9668ae0/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml#L422C14-L422C32